tests/sentry/issues/test_ingest.py

```diff
@@ -31,6 +33,18 @@ def test(self) -> None:
         )
         self.assert_occurrences_identical(occurrence, saved_occurrence)
         assert Group.objects.filter(grouphash__hash=saved_occurrence.fingerprint[0]).exists()
+        # TODO: Query this data and make sure it's present once we have a corresponding dataset in
```

Comment:
Is this blocked on https://github.com/getsentry/snuba/pull/3468?

Reply:
Yeah, once that is merged we'll have the dataset available and can write to it via eventstream and query as well.